src/core_plugins/kibana/public/visualize/editor/editor.js

```diff
@@ -114,19 +129,22 @@ uiModules
       vis: savedVisState
     };
 
-    $state = new AppState(stateDefaults);
```

Comment:
This seemed like a mistake, so to clarify the role of this variable within the enclosure I gave it a unique name and scoped it to the function.